test/unit/lodash.spec.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+            describe('when first param is lower then second param', function ()
+            {
+                it('should return value equal second param', function ()
+                {
+                    var params = datasets.clamp1();
+                    expect(params[1]).to.be.below(params[2]);
+                    expect(_.clamp.apply(_, params)).to.eql(3);
+                });
+            });
+
+            describe('when first param is higher then last param', function ()
+            {
+                it('should return value equal third param', function ()
+                {
+                    var params = datasets.clamp2();
+                    expect(params[2]).to.be.below(params[0]);
```

Comment:
this assertion is ok, but remember that reading documentation is helpful, because you can find there methods that fit better to some situation e.g. method above :)